scripts/build/package.js

```diff
@@ -68,19 +71,38 @@ function minifyAndWrite(filepath, rawJavaScript) {
 }
 
 // 'main'
+const iconsBarrel = [];
 const icons = [];
-data.icons.forEach(icon => {
+data.icons.forEach((icon) => {
   const filename = getIconSlug(icon);
   const svgFilepath = path.resolve(iconsDir, `${filename}.svg`);
-  icon.svg = fs.readFileSync(svgFilepath, UTF8).replace(/\r?\n/, '');
+  icon.svg = fs.readFileSync(svgFilepath, UTF8).replace(/\r?\n/, "");
   icon.slug = filename;
   icons.push(icon);
 
   // write the static .js file for the icon
-  const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
+  const jsFilepath = path.resolve(iconsDir, `${filename}.cjs`);
   minifyAndWrite(jsFilepath, `module.exports=${iconToObject(icon)};`);
+
+  // add name to the barrel file
+  const firstLetter = icon.slug[0].toUpperCase();
+  const rest = icon.slug.slice(1);
+  const capitalizedSlug = `icon${firstLetter}${rest}`;
```

Comment:
The `icon` prefix is necessary here because some icons start with numbers (1Password, for example).

Reply:
Thanks for clarifying - that is a bit unfortunate but I think this makes sense. Going to leave this unresolved to allow others to weigh in though.

Reply:
@simple-icons/maintainers does anyone have any opinions in this regard?

Reply:
From [this comment](https://github.com/simple-icons/simple-icons/pull/6189#issuecomment-903273698):

> There's really nothing we can do about it other than changing the export name, which would cause a mismatch between it and the slug. FontAwesome actually prefixes all their icons with fa or fas (see [docs](https://fontawesome.com/v5.9/how-to-use/with-the-api/setup/importing-icons#icon-names) (granted, they also use it to denote each category [solid, brands, etc]). So I think it would be fine to prefix with "si," like [the react-icons pack for simple-icons](https://react-icons.github.io/react-icons/icons?name=si) does.

As do we in [the font package](https://github.com/simple-icons/simple-icons-font#usage)! Good thing I brought it up I guess. Given the "precedence" that has been set, I think we should use "si" as a prefix rather than "icon". The added benefit is that it prevents any issues if an other icon library decides to use "icon" as a prefix and someone wants to use both.